GithubIssueTagger/Program.cs

```diff
@@ -16,16 +20,90 @@ static async Task Main(string[] args)
                 return;
             }
 
-            var client = new GitHubClient(new ProductHeaderValue("nuget-github-issue-tagger"))
+            _client = new GitHubClient(new ProductHeaderValue("nuget-github-issue-tagger"))
             {
                 Credentials = new Credentials(args[0])
             };
 
-            var unprocessedIssues = await IssueUtilities.GetUnprocessedIssues(client, "nuget", "home");
-            foreach(var issue in unprocessedIssues)
+            await PromptForQuery();
```

Comment:
I don't mind the overall change, but I wouldn't do it by default. 
The default should ideally just complete and not block.

Reply:
I'm confused... there is no default. You have to enter a choice. I can add a default....Which one should be the default? AllUnprocessed?

Reply:
Doesn't it block waiting for input?

Reply:
See if that's what you wanted...

Reply:
yeah, that was the idea.